linux/src/async_socket_linux.c

```diff
@@ -50,6 +54,7 @@ MU_DEFINE_ENUM_STRINGS(ASYNC_SOCKET_SEND_SYNC_RESULT, ASYNC_SOCKET_SEND_SYNC_RES
 #define THREAD_COUNT    1
 
 static volatile_atomic int32_t g_thread_access_cnt;
+static volatile_atomic int32_t g_epoll_access = 0;
 static int g_epoll = -1;
 static THREAD_HANDLE g_thread_array[THREAD_COUNT];
```

Comment:
>THREAD_COUNT [](http://example.com/codeflow?start=36&length=12)

This seems to be just 1. (line 54) and it also seems to be referenced as such for example in `initialize_global_thread`. Note singular "thread".

Reply:
We may have multiple threads depending on perf performance.

Reply:
We will move this to a different module, task has been set